test/libcommon/utility/testutility.cpp

```diff
@@ -299,6 +299,45 @@ void TestUtility::testCurrentVersion() {
     CPPUNIT_ASSERT(std::regex_match(test, std::regex(R"(\d{1,2}\.{1}\d{1,2}\.{1}\d{1,2}\.{1}\d{0,8}$)")));
 }
 
+void TestUtility::testGenerateRandomStringAlphaNum() {
+    {
+        int err = 0;
+        std::set<std::string> results;
+        for (int i = 0; i < 100000; i++) {
+            std::string str = CommonUtility::generateRandomStringAlphaNum();
```

Comment:
The chance are probably insignificant, but since there is no guaranty of uniqueness from method `generateRandomStringAlphaNum`, this test might fail sometimes.